### ISSUE TITLE
IFCWorker.js doesn't work for larger files on firefox

### ISSUE THREAD

**Issue by bartlomiejwiejak** [IFCjs/web-ifc-viewer#224]
Trying to load larger files (more than 268mb) causes error:
`UnknownError: IDBObjectStore.put: The serialized value is too large (size=374079546 bytes, max=267386880 bytes).`

Is there some default for firefox capping it to 267386880 bytes and can we do anything about it? For other browsers it seems to work fine.

**Comment by agviegas** [comment]
Yep, webworkers are not compatible with larger files unfortunately. We are working on a new loading system that is super fast (200mb ifc in 2 seconds), so the webworker approach won't be necessary anymore. For now, disabling webworkers is the only solution for that.
